include/rocksdb/options.h

```diff
@@ -492,6 +492,25 @@ struct DBOptions {
   // Default: false
   bool track_and_verify_wals_in_manifest = false;
 
+  // EXPERIMENTAL: This API/behavior is subject to change
+  // If true, during DB-open try verifying the SST unique id between MANIFEST
+  // and SST properties, which is to make sure the SST is not overwritten or
+  // misplaced. A corruption error will be reported if mismatch detected, but
+  // only when MANIFEST tracks the unique id, which starts from version 7.3.
+  // The user could also the unique id from API
+  // `GetUniqueIdFromTableProperties()`.
+  //
+  // Note:
+  // 1. the option should be used with `max_open_files=-1` to check all SST
```

Comment:
I don't know if we should tie this to max_open_files.

Reply:
Updated to force verifying all SST files regardless `max_open_files`.